examples/triggers/README.md

```diff
@@ -0,0 +1,85 @@
+# Data Hub Framework Flow Example
+
+This example shows you how to integrate Smart Mastering with your Data Hub by using triggers to kick off the mastering process.
+
+## Setup
+
+- Add your username and password to **gradle-local.properties**
+  ```
+  mlUsername=admin
+  mlPassword=admin
+  ```
+- Open a Terminal or Dos prompt into this folder
+- Setup the Example  
+  **\*nix:** `./gradlew setupExample`  
+  **windows:** `gradlew.bat setupExample`  
+
+You have just installed a Data Hub Project into MarkLogic. You also loaded 3 data sets and harmonized them. If none of that makes sense, check out our [Data Hub Framework documentation](https://marklogic.github.io/marklogic-data-hub/).
+
+## Mastering your Data
+
+This example includes a Data Hub Harmonize Flow that will run Smart Mastering against your Harmonized data.
```

Comment:
"This example includes triggers that will run Smart Mastering against your data as it is harmonized."